cfme/tests/infrastructure/test_host.py

```diff
@@ -493,23 +488,14 @@ def test_infrastructure_hosts_navigation_after_download_from_compare(
         initialEstimate: 1/3h
     Bugzilla:
         1747545
-
     """
-    ent_slice = slice(0, num_hosts, None)
-    if hosts_collection == "provider":
-        hosts_view = navigate_to(provider.collections.hosts, "All")
-    elif hosts_collection == "appliance":
-        hosts_view = navigate_to(appliance.collections.hosts, "All")
-    for h in hosts_view.entities.get_all(slice=ent_slice):
-        h.ensure_checked()
-    hosts_view.toolbar.configuration.item_select('Compare Selected items',
-                                                 handle_alert=True)
+    h_coll = locals()[hosts_collection].collections.hosts
+    hosts_view = h_coll.compare_entities(provider, entities_list=h_coll.all()[:num_hosts])
     hosts_view.toolbar.download.item_select(report_format.value)
     if report_format == DownloadOptions.PDF:
         handle_extra_tabs(hosts_view)
     hosts_view.navigation.select("Compute")
-    compare_hosts_view = provider.create_view(HostsCompareView)
-    assert compare_hosts_view.is_displayed
+    assert hosts_view.is_displayed
```

Comment:
Optional: 
Can we include the call to `assert compare_view.verify_checked_items_compared(t_list, compare_view)` here as well for more thorough coverage?

Reply:
Nevermind, I see the method is currently only available for Templates View.